logfire-api/logfire_api/__init__.py

```diff
@@ -303,8 +302,7 @@ def get_baggage(*args, **kwargs) -> dict[str, str]:...
         def set_baggage(*args, **kwargs) -> ContextManager[None]:
             return nullcontext()
 
-        def get_context(*args, **kwargs) -> ContextCarrier:
-            return {}
+        def get_context(*args, **kwargs) -> dict[str, Any]: ...
```

Comment:
Needs to `return {}`. The other functions here which are typed as returning dicts but don't are also wrong and should be fixed.